__mocks__/styleMock.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
👍 nice work figuring out css modules and jest. You can also use identity-obj-proxy which is a bit better if you're doing snapshot testing. But we're not doing snapshot testing so this is 👌

Reply:
:-)